src/plugins/addLoadingFlag.js

```diff
@@ -2,12 +2,13 @@ import { wrap, Chart } from 'highcharts/highstock';
 
 export default () => {
     wrap(Chart.prototype, 'showLoading', function showLoading(proceed, ...args) {
-        this.isLoading = true;
+        if (Object.keys(this).length === 0) return;
```

Comment:
Not sure, thus I did not update package.json

At some point, `this` is a `{}` when `showLoading` is called
And I do not know why, any clue from your side?